app/controllers/patient/registrations_controller.rb

```diff
@@ -11,4 +12,8 @@ def configure_sign_up_params
     devise_parameter_sanitizer.permit(:sign_up, keys: [:name, :gender, :age, :phone, :country, :birth])
   end
 
+  def configure_account_update_params
+    devise_parameter_sanitizer.permit(:account_update) { |u| u.permit(:name, :gender, :phone, :country, :birth, :email, :description, :password, :password_confirmation, :current_password) }
```

Comment:
This line is too long. Please consider using a multiline expression.

For example:

```ruby
devise_parameter_sanitizer.permit(:account_update) do |u| 
  u.permit(
    :name, 
    :gender, 
    :phone,
    # ...
  )
end
```